### DISCUSSION TITLE
Question about timestep size in LAMMPS with deepmd models

### DISCUSSION THREAD

**Discussion by ramic-k** [deepmodeling/deepmd-kit#1884] [General]
Hi, I have a general question about what I could possibly be doing wrong when using DeepMD. I have trained few models for different materials using VASP AIMD and different configurations and temperatures with energy and forces converged, but one thing that is consistent is that when I use these models in LAMMPS MD simulation, the largest timestep I can choose is lets say 0.1fs or so. Is that normal, or I am doing something wrong? In general the training data was created using NVT ensamble and 1fs time step, but in some cases I tried to vary those as well, but LAMMPS MD step would still need to be 0.1fs or less, or otherwise the simulation blows up.

**Comment by wanghan-iapcm** [comment]
Are you running DPMD simulations in exactly the same thermodynamic conditions as the AIMDs?  How about the accuracy of your model?
